app/controllers/application_controller.rb

```diff
@@ -3,6 +3,11 @@ class ApplicationController < ActionController::Base
 
   include RealIpLogger
 
+  # Error modal contact message
+  ZENDESK_CONTACT = "If this error persists and you require assistance, please contact support at " \
```

Comment:
IMO the 'and you require assistance' is redundant, but that's a good topic for demo

Reply:
Also, I'd name this constant SCP_SUPPORT_EMAIL_MESSAGE, since it's not just the contact, and who knows when zendesk will get replaced by <<new DSP tech>>

Reply:
390a54e